ci/build.sh

```diff
@@ -18,6 +18,39 @@ function get_branch_to_checkout() {
     || echo "${TRAVIS_BRANCH}"
 }
 
+function bundle_fluentd_plugins() {
+  local VERSION="${1}"
+
+  if [[ "${VERSION}" == "" ]] ; then
+    echo "Please provide the version when bundling fluentd plugins"
+    exit 1
+  fi
```

Comment:
Does this check do anything? Is it possible to not pass VERSION as we set it up in header?

Reply:
With the current `VERSION` variable declaration it doesn't really do anything. Better safe than sorry. I can remove it if don't want this here

Reply:
Just wanted to ensure

Reply:
It's good to know if version is there.